### ISSUE TITLE
Unable to install via pip on python 3.10

### ISSUE THREAD

**Issue by JacobCallahan** [ParallelSSH/ssh2-python#160]
### Bug reports
I am unable to install ssh2-python on systems with Python 3.10.

__Steps to reproduce:__

1. Be on a python 3.10 system (potentially linux only)
2. `pip install ssh2-python`

__Expected behaviour:__ ssh2-python installs successfully like in previous python versions.

__Actual behaviour:__ the package fails to build, see below for full output

__Additional info:__

Reproducible on:
- Fedora 35 with Python 3.10
- Offcial Python 3.10 container image

```
Collecting ssh2-python
  Downloading ssh2-python-0.26.0.tar.gz (1.1 MB)
     |████████████████████████████████| 1.1 MB 3.1 MB/s 
Building wheels for collected packages: ssh2-python
  Building wheel for ssh2-python (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-tss1xwng
       cwd: /tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/
  Complete output (10 lines):
  /bin/sh: 1: cmake: not found
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py", line 34, in <module>
      build_ssh2()
    File "/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/_setup_libssh2.py", line 21, in build_ssh2
      check_call('cmake ../libssh2 -DBUILD_SHARED_LIBS=ON \
    File "/usr/local/lib/python3.10/subprocess.py", line 369, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command 'cmake ../libssh2 -DBUILD_SHARED_LIBS=ON     -DENABLE_ZLIB_COMPRESSION=ON -DENABLE_CRYPT_NONE=ON     -DENABLE_MAC_NONE=ON -DCRYPTO_BACKEND=OpenSSL' returned non-zero exit status 127.
  ----------------------------------------
  ERROR: Failed building wheel for ssh2-python
  Running setup.py clean for ssh2-python
Failed to build ssh2-python
Installing collected packages: ssh2-python
    Running setup.py install for ssh2-python ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-tbnhbts_/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.10/ssh2-python
         cwd: /tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/
    Complete output (10 lines):
    /bin/sh: 1: cmake: not found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py", line 34, in <module>
        build_ssh2()
      File "/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/_setup_libssh2.py", line 21, in build_ssh2
        check_call('cmake ../libssh2 -DBUILD_SHARED_LIBS=ON \
      File "/usr/local/lib/python3.10/subprocess.py", line 369, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command 'cmake ../libssh2 -DBUILD_SHARED_LIBS=ON     -DENABLE_ZLIB_COMPRESSION=ON -DENABLE_CRYPT_NONE=ON     -DENABLE_MAC_NONE=ON -DCRYPTO_BACKEND=OpenSSL' returned non-zero exit status 127.
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xexeheh5/ssh2-python_f5296b320aa64158990e2ca6ce84db8a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-tbnhbts_/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.10/ssh2-python Check the logs for full command output.
```

**Comment by pkittenis** [comment]
Resolved in master, pending new release.

**Comment by pkittenis** [comment]
Resolved by `0.27.0`
